docs/netlify.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+  ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../docs"
```

Comment:
Wouldn't this work the same as it currently works? Currently, Netlify checks if there are changes in `docs` and cancels the build if there are no changes.

Reply:
Strange point, I read the documentation for Netlify, and it seems that you need to configure this yourself, and this is what I tested in my replica repository.

----

It looks like it only needs to be adjusted for cases where bot comments are needed.